test/gurka/test_reproduce_tile_build.cc

```diff
@@ -85,17 +88,44 @@ void assert_tile_equalish(const GraphTile& a, const GraphTile& b) {
   }
 }
 
+uint64_t get_pbf_md5(std::string pbf_path) {
```

Comment:
same same but different. basically a half-independent md5 calculation to get a more robust test